csharp/dotnet-windows/console/Samples/Samples/Program.cs

```diff
@@ -238,7 +235,7 @@ public static async Task GetProsodyResult()
             var audioConfig = AudioConfig.FromWavFileInput(wavePath);
 
             // create pronunciation assessment config, set grading system, granularity and if enable miscue based on your requirement.
-            string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":true, \"enableProsodyAssessment\":\"" + enableProsodyAssessment + "\"}";
+            string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":true, \"enableProsodyAssessment\":\"true\"}";
```

Comment:
For other configure, the first letter is capitalized. So please change `enableProsodyAssessment` to `EnableProsodyAssessment`.

Reply:
udpated